src/awslambda/extract/common_utils.py

```
@@ -578,7 +579,20 @@ def save_s3_bucket_by_parquet(
     except Exception as e:
         print(f"[ERROR] S3 업로드 실패: {str(e)}")
         return None    
-   
+
+def get_my_ip():
+    try:
+        # Option 1: Using ipify API
+        response = requests.get('https://api.ipify.org')
+        print(f"[INFO] AWS NAT Gateway 변환 이후 IP: {response.text.strip()}")
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)